tests/clients/test_experiment.py

```diff
@@ -510,3 +510,15 @@ def test_log_run_data_empty(mocker):
 
     client.log_run_data(PROJECT_ID, EXPERIMENT_RUN_ID)
     ExperimentClient._patch_raw.assert_not_called()
+
+
+def test_delete(mocker):
+    mocker.patch.object(ExperimentClient, "_delete_raw")
+
+    client = ExperimentClient(mocker.Mock())
+    result = client.delete(PROJECT_ID, EXPERIMENT_ID)
+
+    assert result is None
```

Comment:
Similar to my comment on documentation, we'd not typically assert that the return value of a method in this case.